README.rst

```diff
@@ -69,6 +69,20 @@ FogLAMP is currently based on C/C++ and Python code. The packages needed to buil
 - sqlite3
 - libsqlite3-dev
 
+
+Linux distributions
+-------------------
+
+FogLAMP can be build or installed in one of the following Linux distributions :
+
+- Ubuntu
+- Raspbian
+- Red Hat
+- CentOS
```

Comment:
Also Red Hat 7.6 and SentOS 7.6 - we should be clear we are not including CentOS or RHEL 8

Reply:
done